ban/http/api.py

```diff
@@ -203,15 +203,33 @@ def get_resource(self, identifier):
     @app.jsonify
     @app.endpoint('/<identifier>', methods=['POST'])
     def post_resource(self, identifier):
-        """Patch {resource} with 'identifier'.
+        """Post {resource} with 'identifier'.
 
         parameters:
             - $ref: '#/parameters/identifier'
+              name: identifier
+              in: path
+              type: string
+              required: true
+              description: {resource} identifier
+            - name: body
```

Comment:
In this case, 'body' represents json text to give to create a post_resource.
May be we can rename this parameter fom 'body' to 'json'?